src/XRP/model/xrp-transaction.ts

```diff
@@ -162,12 +155,10 @@ export default class XRPTransaction {
 
   /**
    * @param hash The identifying hash of the transaction.
-   * @param account The unique address of the account that initiated the transaction.
+   * @deprecated @param account Deprecated, please use sourceXAddress. The unique address of the account that initiated the transaction.
```

Comment:
Misc: This is why when you start to get over ~3 parameters, it often makes sense to switch the call signature to a single parameter that is an object of properties.

Because right now, you have a deprecated parameter in the constructor function that _must_ be provided, because the order of parameters matters. If we used an object here, we could just omit the `account` property.